src/rest_framework_jwt/utils.py

```diff
@@ -68,6 +69,7 @@ def jwt_create_payload(user):
     expiration_time = issued_at_time + api_settings.JWT_EXPIRATION_DELTA
 
     payload = {
+        'jti': uuid.uuid4(),
```

Comment:
As adding >40 bytes to each and every jwt (or potentially double that for `orig_jti`) is not exactly a lightweight change, I would prefer to make this configurable. Why not, for example, instead of `JWT_REQUIRE_TOKEN_ID` have `JWT_TOKEN_ID` with values `off`, `add` or `require`?

Reply:
Makes sense.

I've slimmed them down in a couple of ways.

- (0de4ce5e0262a1b7ee08bd52cee70699cec70d50) we don't really need `orig_jti` in the original token, and since a UUID is fairly hefty I've left it out. I think it's reasonable to read from either orig_jti or jti as needed
- (8e0bb174345c8e97eea5ed957f6da62dfed0d36a) I've implemented a three-state config, along the lines you suggest; thanks for the suggestion